modules/index/src/integrationTest/java/org/apache/ignite/internal/index/ItIndexNodeFinishedRwTransactionsCheckerTest.java

```diff
@@ -74,8 +74,8 @@ protected int initialNodes() {
     @BeforeEach
     void setUp() {
         if (node() != null) {
```

Comment:
Why we need this check? If node is null, is it a valid state to start a test? 

I removed checks from tearDown / beforeEach and works like a charm

Reply:
I also don’t see the point in checking, I removed it, thanks!